content.js

```diff
@@ -0,0 +1,288 @@
+function detectStoryAndAppendTimerElement() {
+    const shortcutStoryModalContainerId = "story-dialog-parent";
+
+    const storyModal = document.getElementById(shortcutStoryModalContainerId);
+
+    if(!storyModal) {
+        console.log("Story modal not found");
+        return;
+    }
+
+    const harvestTimerID = "polaris-harvest-timer";
+
+    if(storyModal) {
+        // chrome.runtime.sendMessage({ action: 'checkIfHarvestTimerIsRunning' }, function(response) {
+        // });
+
+        const {
+            storyId,
+            storyName,
+            storyPermaLink
+        } = getShortcutStoryDetails();
+
+        const elementToAppendId = document.getElementById('cid-breadcrumbs-story-dialog');
+        const startTimerButton = document.getElementById(harvestTimerID);
+
+        if(elementToAppendId) {
+            
+            console.log("Element found to append timer.");
+            if(!storyId || !storyName || !storyPermaLink) {
+                console.log("Story details not found.");
+                return;
+            }
+
+            createStartTimerButton("shortcut", elementToAppendId, harvestTimerID, storyId, storyName, storyPermaLink);
+
+            window._harvestPlatformConfig = {
+                "applicationName": "Crowdlinker",
+                "permalink": window.location.href,
+                "skipStyling": true // Override the default styling
+            };
+
+            // Use to dispatch event after loading buttons
+            const event = new CustomEvent("harvest-event:timers:add", {
+                detail: { element: document.querySelector(".harvest-timer") }
+            });
+            document.querySelector("#harvest-messaging").dispatchEvent(event);
+        }
+    }
+}
+
+function detectGoogleCalendarEvent() {
+    const googleCalendarEventContainer = document.querySelector('[data-open-edit-note]');
+    const eventHeaderElement = document.querySelector('.wv9rPe');
+
+    if(!googleCalendarEventContainer || !eventHeaderElement) {
+        return;
+    }
+
+    const harvestTimerID = "polaris-harvest-timer";
+
+    if(!document.getElementById(harvestTimerID)) {
+        const eventName = document.querySelector('[data-open-edit-note] [data-text]').textContent;
+        const eventId = googleCalendarEventContainer.getAttribute('data-eventid');
+
+        createStartTimerButton("google-calendar", eventHeaderElement, harvestTimerID, eventId, eventName);
+
+        window._harvestPlatformConfig = {
+            "applicationName": "Crowdlinker",
```

Comment:
@mayank2424 @ankesh7 Can we move these to a common `config.js` so it's easier for other open source users to modify for their own needs?

Same for any RGB/Hex colors that you feel will be nice to haves for the community.